x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_context_menu.tsx

```diff
@@ -88,8 +97,5 @@ const AbsoluteWrapper = euiStyled.div`
 `;
 
 const ButtonWrapper = euiStyled.div`
-  background: ${(props) => props.theme.eui.euiColorPrimary};
-  border-radius: 50%;
-  padding: 4px;
-  transform: translateY(-6px);
+  transform: translate(-6px, -6px);
```

Comment:
As Katrin pointed out, the button shifts to the right when you interact with it. It's because EuiButton does `transform: translateY(-1px)` for :active and :hover (shown here https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_button.scss). We'll need to add some CSS that overrides the :active and :hover states for EuiButton. We'll need something like `transform: translate(-6px, -7px)` on :active and :hover for this button

Reply:
Did you pull the last commit? The issue that Katrin reported was fixed there.

Previously it was happening because I was overriding the `transform` of the button itself, but only in the "resting" state. The last commit applies the transform to the container.

There _is_ a certain movement in the button by default, which maybe is what you're seeing. You can test this deleting the line where you have commented.